sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Generated/SdkInfo_SqlManagementClient.cs

```diff
@@ -20,7 +20,7 @@ internal static partial class SdkInfo
               return new Tuple<string, string, string>[]
               {
                 new Tuple<string, string, string>("Sql", "BackupLongTermRetentionPolicies", "2017-03-01-preview"),
-                new Tuple<string, string, string>("Sql", "BackupShortTermRetentionPolicies", "2017-10-01-preview"),
+                new Tuple<string, string, string>("Sql", "BackupShortTermRetentionPolicies", "2020-02-02-preview"),
```

Comment:
`AzSdk.RP.props` needs to be updated whenever you change API version(s)

Reply:
Where is that? These are auto generated files so maybe when I generate via swagger master branch rather than feature branch can make that file generated.

Reply:
Here: it's written by hand
https://github.com/Azure/azure-sdk-for-net/blob/5a96eb4dd13ebb56102d2292dd2819bbad859973/sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/AzSdk.RP.props

Reply:
ACK, will update in next push.

Reply:
Synced with master, the V2020-02-02 already added into AzSdk.PR.props.